### ISSUE TITLE
Use SCSS as default style sheet language

### ISSUE THREAD

**Issue by michaeljota** [hexojs/hexo#3032]
## Environment Info

Node version(`node -v`): 8.7.0

Hexo and Plugin version(`npm ls --depth 0`): 
```bash
+-- hexo@3.5.0
+-- hexo-deployer-git@0.3.1
+-- hexo-generator-archive@0.1.5
+-- hexo-generator-category@0.1.3
+-- hexo-generator-feed@1.2.2
+-- hexo-generator-index@0.2.1
+-- hexo-generator-tag@0.2.0
+-- hexo-migrator-rss@0.1.4
+-- hexo-renderer-ejs@0.3.1
+-- hexo-renderer-marked@0.3.2
+-- hexo-renderer-scss@1.1.0
`-- hexo-server@0.2.2
```
## For feature request

Maybe when Stylus was selected as the default style sheet language it was somehow popular or something, but now SCSS is used more than plain CSS ([ref](https://stateofjs.com/2017/css/results)). 

SCSS is used by the most popular CSS Frameworks, such Bootstrap 4, Materialize, and Fundation, so used by default would allow theme developers to use the files of such frameworks to develop better themes.

I know there is a SCSS plugin, I'm using it, but if SCSS is included by default, users won't need to install any plugins.

**Comment by lucianobarauna** [comment]
I agree with your remark and also includes that the stylus is staffless to keep the project has a time without pull request and with it it is falling into disuse
